src/web/app/pages-help/instructor-help-page/instructor-help-students-section/instructor-help-students-section.component.html

```diff
@@ -147,18 +147,50 @@ <h4 *ngIf="displaySubsection(3, 5)">Finding Students</h4>
                             [id]="StudentsSectionQuestions.STUDENT_EMAIL"
                             headerText="How do I email a student or students in my course?"
                             [(isPanelExpanded)]="questionsToCollapsed[StudentsSectionQuestions.STUDENT_EMAIL]">
+    <p>If your course has a limited number of students and you want to send email to one or some of them:</p>
     <ol>
       <li>
         On the <b>Students</b> page, filter out the selected student/students.
       </li>
       <li>
-        Click
-        <button class="btn btn-sm btn-secondary">Copy Emails</button> to copy their email addresses to your clipboard.
+        Use mouse to select students' email addresses, and copy them to your clipboard.
       </li>
       <li>
         Use your preferred email provider to email the students.
       </li>
     </ol>
```

Comment:
I think we could be clearer with the word "filter". Filter might refer to the old students page where there was probably an actual option to filter the list of students, at least that's what the word leads me to believe. Debatable though. How about 

```
1. On the <b>Students</b> page, select the students' email addresses and copy them to your clipboard.
2. Use your preferred email provider to email the students.
```

What do you think?

Reply:
Thanks for the comment. Sounds good for me.

I also thought for a while when I edited this part. I left it there as I felt it can also mean using the sorting headers to find the student desired. However, what you mention is a valid concern and I will edit it accordingly.

Reply:
This has been fixed as per your comment.

Reply:
There's an added [comment](https://github.com/TEAMMATES/teammates/issues/11547#issuecomment-1030882558) on the original issue. Should we rephrase it to something along the lines of "how can i get the email addresses of my students"?

Reply:
Ok I guess we can put this on hold first until the original issue has a conclusion.